docs/Installation.md

```diff
@@ -39,22 +39,31 @@ The `Project` subdirectory contains many [example environments](Learning-Environ
 to help you get started.
 
 ### Package Installation
+ML-Agents C# SDK is transitioning to a Unity Package.  While we are working on getting into the
+official packages list, you can add the `com.unity.ml-agents` package to your project by
+navigating to the menu `Window`  -> `Package Manager`.  In the package manager window click
+on the `+` button.
 
-If you intend to copy the `com.unity.ml-agents` folder in to your project, ensure that
-you have the [Barracuda preview package](https://docs.unity3d.com/Packages/com.unity.barracuda@0.3/manual/index.html) installed.
+<p align="center">
+  <img src="images/unity_package_manager_window.png"
+       alt="Linux Build Support"
+       width="500" border="10" />
+</p>
+**NOTE:** In Unity 2018.4 it's on the bottom left and in Unity 2019.3 it's on the top left.
 
-To install the Barracuda package in later versions of Unity, navigate to the Package
-Manager window by navigating to the menu `Window` -> `Package Manager`.  Click on the
-`Advanced` dropdown menu to the left of the search bar and make sure "Show Preview Packages"
-is checked.  Search for or select the `Barracuda` package and install the latest version.
+Select `Add package from disk...` and navigate into the
+`com.unity.ml-agents` folder and select the `package.json` folder.
 
 <p align="center">
-  <img src="images/barracuda-package.png"
-       alt="Barracuda Package Manager"
-       width="710" border="10"
-       height="569" />
+  <img src="images/unity_package_json.png"
+       alt="Linux Build Support"
+       width="500" border="10" />
 </p>
 
+If you are going to follow the examples from our documentation, you can open the `Project`
+folder in Unity and start tinkering immediately.
+
```

Comment:
There are some project settings that the user should modify in the PlayerSettings no ?

Reply:
Like what?